.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,20 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue with .NET runtime or core .NET libraries
+    url:  https://github.com/dotnet/runtime/issues/new/choose
+    about: Please open issues with the .NET runtime or core .NET libraries in their repo
+  - name: Issue with .NET Core SDK
+    url:  https://github.com/dotnet/sdk/issues/new/choose
+    about: Please open issues with the .NET Core SDK in their repo
+  - name: Issue with Entity Framework
```

Comment:
Should this be `Entity Framework Core` instead?
Or maybe, we should even have two separate ones for EF and EF Core?

@ajcvickers thoughts?

Reply:
I changed to Entity Framework Core. I can add regular EF, if you let me know the URL.

Reply:
@ajcvickers?

Reply:
@danmosemsft Looks good the way it is now.